aperturedb/Status.py

```diff
@@ -132,6 +136,50 @@ def count_connections(self, connections_class, constraints={}):
             total_connections = res[0]["FindConnection"]["count"]
         except:
             total_connections = 0
-            print(self.connector.get_last_response_str())
+            self.connector.print_last_response()
 
         return total_connections
+
+    def count_descriptorsets(self):
```

Comment:
Is this to be removed ? Not invoked from anywhere.

Reply:
these are used by clients. I added because I use it quite often. I think we should rename Status altogether to "Utils" or something of the kind. Status is more than just showing the status now. We can have Utils that have many utility methods, together with showing the status, what do you think?

Reply:
This can go into Utils, or even a class like Images for [DescriptorSets](https://docs.aperturedata.io/commands/AddDescriptorSet.html) OM .

Reply:
cool sounds good, let me do some refactoring then, and I need to make sure those tests pass!

Reply:
done! added more stuff to Utils as well.